packages/yourloops/lib/team/hook.tsx

```diff
@@ -63,6 +63,7 @@ export function iMemberToMember(iTeamMember: ITeamMember, team: Team, users: Map
     idVerified,
     alarms,
     monitoring,
+    unreadMessages,
```

Comment:
As unreadMessages is stored in the teams context , it should be updated when message are read ? (messages are marked as read when they are loaded in the dashboard).